srcpkgs/libspnav/template

```diff
@@ -1,24 +1,20 @@
 # Template file for 'libspnav'
 pkgname=libspnav
-version=1.0
-revision=2
+version=1.1
+revision=1
 build_style=gnu-configure
-configure_args="--disable-opt"
 makedepends="libX11-devel"
 short_desc="Open source alternative to 3DConnextion drivers"
-maintainer="yopito <pierre.bourgin@free.fr>"
```

Comment:
don't adopt non-orphaned packages without asking the current maintainer (cc @yopito)

also, I'd prefer to see more contributions to the package before adopting it

Reply:
I tried to discuss this in an issue (https://github.com/void-linux/void-packages/issues/44550) which you closed without feedback. Granted I used the wrong issue template it seems but we're talking about a few lines here so absent any guidance from you I figured we could discuss it in a PR.

I'll force push this branch in a minute to revert this. I tried to guess based on PRs you reviewed whether I'd have to adopt it for this PR to be approved and I was aiming for approval on the first review :).

Reply:
Reverted maintainer in https://github.com/void-linux/void-packages/commit/40656499e6869801d70e022b3e040c1b7ebe8cef